6-AdvancedScenarios/4-hybrid-SPA/App/client/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/msal-browser": "^2.22.1",
+    "@azure/msal-react": "^1.3.1",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.4",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
Testing libraries should be development dependencies.

Reply:
@jasonnutter libraries moved to development dependencies